src/App.js

```diff
@@ -1,14 +1,31 @@
 import React from 'react';
-import Header from './components/Header/Header';
-import styles from './App.module.scss';
+import { Redirect, Route, Switch } from 'react-router-dom';
+import HeaderContainer from './components/Header/HeaderContainer';
 import Board from './components/Board/Board';
+import AuthContainer from './components/Auth/AuthContainer';
+import styles from './App.module.scss';
 
-function App() {
+const App = ({ isLogged }) => {
   return (
-    <div className={styles.app}>
-      <Header />
-      <Board />
-    </div>
+      <>
+        <Switch>
+          <Route exact path="/">
+            {
+              isLogged
+                ? (
+                  <div className={styles.app}>
+                    <HeaderContainer />
+                    <Board />
+                  </div>
+                )
+                : <Redirect to="/auth" />
+            }
+          </Route>
```

Comment:
When you want to make some components available only for authenticated users you should create a route guard